src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java

```diff
@@ -14,27 +14,26 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public class HyperwalletUser extends HyperwalletBaseMonitor {
 
-    public static enum Gender {MALE, FEMALE}
 
-    public static enum ProfileType {INDIVIDUAL, BUSINESS, UNKNOWN}
+    public enum Gender {MALE, FEMALE}
```

Comment:
Why are there changes to the existing enums like Gendre, ProfileType?
Why are they no longer static?